Content.Server/Mining/Electromagnet.cs

```diff
@@ -0,0 +1,79 @@
+using Content.Server.Power.Components;
+using Content.Shared.Tag;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Systems;
+
+namespace Content.Server.Mining;
+
+[RegisterComponent]
+public class ElectromagnetComponent : Component
+{
+    [DataField("range")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float Range = 2f;
+
+    [DataField("force")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float Force = 300f;
+
+    [DataField("minDist")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float MinDist = 0.5f;
+}
```

Comment:
This should bet networked sealed and shared.

Reply:
How come? It only makes sense for the server to be running the physics update for the electromagnet. What does the client need the Electromagnet component for?

Reply:
All components should be shared unless there's good reason not to, plus the client should be predicting this for its own entity or otherwise predicted physics bodies.